src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './NewTodo.css';
+
+class NewTodo extends React.Component {
+  state = {
+    title: '',
+    name: '',
+    completed: '',
+    userId: '',
+    id: this.props.todoId,
+    status: '',
+    showTitleError: false,
+    showUserError: false,
+    showStatusError: false,
+  }
+
+  addTodoText = (event) => {
+    this.setState(
+      {
+        title: event.target.value.replace(/[^a-zа-я0-9\s]/g, ''),
+        showTitleError: false,
+      },
+    );
+  }
+
+  addUser = (id) => {
+    const { users } = this.props;
+
+    this.setState(
+      {
+        userId: +id,
+        name: users.find(user => user.id === +id).name,
+        showUserError: false,
+      },
+    );
+  }
+
+  statusTodo = (event) => {
+    const target = event.target.value;
```

Comment:
```suggestion
    const value = event.target.value;
```
`value` is a more logical option in this case